backend/contracts/ExpenseTracker.sol

```diff
@@ -45,8 +46,10 @@ contract ExpenseTracker {
         tracker[msg.sender].total += 1;
         tracker[msg.sender].transactions.push(_id);
         all_ids[_id] = true;
+        emit addexpense(_id,_amount, _description, _category, _date);
     }
 
+    event addincome(string id,uint amount,string description,string category,string date);

```

Comment:
please move all the event declarations at the top

Reply:
ok